### ISSUE TITLE
Copy incorrectly if my text contains "\r\n" when using firefox

### ISSUE THREAD

**Issue by LiuRhoRamen** [sudodoki/copy-to-clipboard#124]
I need to copy text which contains "\r\n". when using firefox, I found that the "\r\n" inside the copied text turned to be "\n\n".
After reading the source code, I found the problem is the use of `Selection`. (Maybe it's a bug of Firefox)

Heres a simple example to explain the problem:
https://jsfiddle.net/LiuRhoRamen/mo1zfxvb/7/

Maybe It's better to create a `textarea`, call .select() method and do the copy？

**Comment by sudodoki** [comment]
yes, that's an issue I have observed as well. Using textarea introduces other issues. Did you try using alternative copy packages?

**Comment by LiuRhoRamen** [comment]
Yeah, I've found [sindresorhus/copy-text-to-clipboard](https://github.com/sindresorhus/copy-text-to-clipboard) which use textarea, but I'm not sure if it will cause some other issues that you mentioned.
Also I've found there's a new api called navigator.clipboard that can be used in modern browsers.

**Comment by A-ZHANG1** [comment]
For plain/text, try replace /r/n with ES6/Babel [backticks](https://stackoverflow.com/questions/805107/creating-multiline-strings-in-javascript).i.e.
`
text.replace(/(\r\n)/g, `
`);
`
For html, the solution is to create [<textarea>](https://jsfiddle.net/andmetoo/p02ump42/6/).
